doc/nrf/app_dev/device_guides/nrf54l/cryptography.rst

```diff
@@ -0,0 +1,336 @@
+.. _ug_nrf54l_cryptography:
+
+nRF54L Series cryptography
```

Comment:
Please add link to this  page in the table on https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/security.rst

Not sure if this should be a separate row under Cryptographic operations or an addition to that row.

Reply:
Link added in PR https://github.com/nrfconnect/sdk-nrf/pull/17804, a draft PR with work in progress.

Not sure if it should be a separate row either...
I added it along with the Crypto operations entry for the moment, what do you think?
If we add similar content for the nRF54H Series devices in time for the release, we should have own entry... Product-specific cryptography operations... or something like that.